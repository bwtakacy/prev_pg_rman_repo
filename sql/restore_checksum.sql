\! bash sql/restore.sh "--with-checksum"

