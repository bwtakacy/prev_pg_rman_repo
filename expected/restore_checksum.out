\! bash sql/restore.sh "--with-checksum"
###### RESTORE COMMAND TEST-0001 ######
###### recovery to latest from full backup ######
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
0

###### RESTORE COMMAND TEST-0002 ######
###### recovery to latest from full + incremental backups ######
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
0

###### RESTORE COMMAND TEST-0003 ######
###### recovery to latest from compressed full backup ######
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
0

###### RESTORE COMMAND TEST-0004 ######
###### recovery to latest from full + archivelog backups ######
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
0
0

###### RESTORE COMMAND TEST-0005 ######
###### recovery to target timeline ######
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
0
0
checking recovery.conf...
OK: recovery.conf has the given target timeline.

###### RESTORE COMMAND TEST-0006 ######
###### recovery to target time ######
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
0

###### RESTORE COMMAND TEST-0007 ######
###### recovery to target XID ######
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
0
OK: recovery-target-xid options works well.

###### RESTORE COMMAND TEST-0008 ######
###### recovery with target inclusive false ######
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
0
OK: recovery-target-inclusive=false works well.

###### RESTORE COMMAND TEST-0009 ######
###### recovery with hard-copy option ######
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
0
OK: hard-copy option works well.

###### RESTORE COMMAND TEST-0010 ######
###### recovery from incremental backup after database creation ######
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
0

